src/main/java/com/sonymobile/jenkins/plugins/mq/mqnotifier/MQNotifierConfig.java

```diff
@@ -52,7 +51,8 @@
  * @author Örjan Percy &lt;orjan.percy@sonymobile.com&gt;
  */
 @Extension
-public final class MQNotifierConfig extends Plugin implements Describable<MQNotifierConfig> {
+@Symbol("mqNotifier")
```

Comment:
According to https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/PLUGINS.md#step-6 this should preferably be the plugin id, i.e. mq-notifier.

Reply:
True, I'll correct it.